### PR TITLE
fuzzaldrin-plus v0.6.0

### PR DESCRIPTION
**Pull request by reyronald**
Flow library definitions for `fuzzaldrin-plus` npm package.

/cc @jeancroy

---

- [x] Add definitions & tests for previous versions of Flow (0.25 onwards)
- [x] Make the following test pass:

```js
// $ExpectError
fuzzaldrinPlus.filter([{ title: "Maybe" }, { title: "Me" }], "me", {
    incorrectProperty: true, 
    // ^^^^^^^^^^^^^^^^^^^^^ This should error because `incorrectProperty` is an unknown property
    key: "title"
});
```

### CODE REVIEW

File: definitions/npm/fuzzaldrin-plus_v0.6.0/flow_v0.76.x/fuzzaldrin-plus_v0.6.0.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// @flow
```
Comment (GAntoine) [inline definitions/npm/fuzzaldrin-plus_v0.6.0/flow_v0.76.x/fuzzaldrin-plus_v0.6.0.js]: Would recommend changing `flow_v0.76.x` to `flow_v0.76.x-` (notice the `-`).

File: definitions/npm/fuzzaldrin-plus_v0.6.0/flow_v0.76.x/fuzzaldrin-plus_v0.6.0.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+// Flow-typed type definitions for fuzzaldrin-plus v0.6.0
+// Project: https://github.com/jeancroy/fuzz-aldrin-plus
+// Author: Jean Christophe Roy <https://github.com/jeancroy>
+// Definitions by: Ronald Rey <https://github.com/reyronald>
```
Comment (GAntoine) [inline definitions/npm/fuzzaldrin-plus_v0.6.0/flow_v0.76.x/fuzzaldrin-plus_v0.6.0.js]: You can create a `.md` file in `/npm/fuzzaldrin-plus_v0.6.0` with any info you want future contributors to know. Something like an `AUTHORS.md` or `CONTRIBUTING.md`.

**Comment by reyronald** [comment]
Hello @GAntoine , thank you for the review!

 I pushed the suggested changes and fixed some other things that were pending. **This is ready for another review and potential merge**!

**Comment by arahansen** [comment]
I'm seeing issues running `flow-typed install` because of the `CONTRIBUTING.md` file included here. Is there a way to work around this?

**Comment by reyronald** [comment]
Yes, update `flow-typed`

**Pull request merged**